pact-consumer-api/pom.xml

```diff
@@ -54,6 +61,15 @@
 				<groupId>org.springframework.boot</groupId>
 				<artifactId>spring-boot-maven-plugin</artifactId>
 			</plugin>
+			<plugin>
```

Comment:
Configuring the PACT contract JSON output directory from `target/pacts` -> `src/pacts`